content/java/tutorials/adding-new-tactical-symbols.md

```diff
@@ -56,91 +56,16 @@ The new icons should be white and black on a transparent background. WorldWind w
 
 The following table shows the all eight icons for the Movement Control Center (MCC) Theater graphic, and how the final symbol will appear when created with and without a frame. Notice that the icon for each standard identity is slightly different so that each icon matches its frame exactly. Also notice that the white pixels of the icon are converted to black when the icon is drawn on a frame, and to the standard identity color when the icon is drawn without a frame. The anticipated version of the icon is only used to create an unframed symbol. For an anticipated framed symbol the frame itself is drawn with a dashed border.
 
```

Comment:
Great simplification. I was wondering why we were using HTML tables in markdown. Do this make the fix-tables shortcode obsolete?